src/main/java/org/jfrog/bamboo/release/vcs/AbstractVcsManager.java

```diff
@@ -44,10 +45,14 @@ protected BuildContext getContext() {
     @Nullable
     protected Map<String, String> getTaskConfiguration() {
         List<TaskDefinition> tasks = this.getContext().getBuildDefinition().getTaskDefinitions();
-        for (int i = 0; i < tasks.size(); i++) {
-            if (tasks.get(i) != null &&
-                    tasks.get(i).getPluginKey().contains("bamboo-artifactory-plugin")) {
-                return tasks.get(i).getConfiguration();
+        for (TaskDefinition taskDefinition : tasks) {
+            // Check that task definition enabled.
+            if (taskDefinition.isEnabled()) {
+                AbstractBuildContext config = AbstractBuildContext.createContextFromMap(taskDefinition.getConfiguration());
+                // Check the release management is enabled
+                if ((config != null) && config.releaseManagementContext.isReleaseMgmtEnabled()) {
```

Comment:
Redundant parentheses around ```(config != null)```.

Reply:
Per our discussion, the config can be null so this check is not redundant.